produce_snp_matrix.py

```diff
@@ -0,0 +1,44 @@
+from glob import glob
+import subprocess
+import argparse
+import os
+
+def snps(input_path, output_path):
+    # glob filenames
+    megafasta = ""
+    paths = glob(input_path + "*_consensus.fas")
+    for path in paths:
+        file = open(path)
+        fasta = file.read()
+        megafasta = megafasta + fasta + '\n'
+
+    if os.path.exists(output_path + 'output'):
+        os.chdir(output_path + 'output')
+    else:
+        os.mkdir(output_path + 'output')
+        os.chdir(output_path + 'output')
```

Comment:
Ah, so maybe it would be better not to change directories to here and instead just give the entire filepaths to the run commands on lines 27 and 31